src/main/steps/applicant1/hub-page/content.ts

```diff
@@ -28,6 +28,11 @@ const en = ({ isDivorce, userCase, referenceNumber, partner, isJointApplication,
     applyForConditionalOrder: 'Apply for conditional order',
     applyForFinalOrder: 'Apply for final order',
   },
+  awaitingHwfPartPayment: {
+    line1: 'A decision has been made on your Help with Fees application.',
+    line2:
+      'The court will contact you with details about the decision on your Help With Fees application and what you need to do next.',
```

Comment:
NitPick: line 2 has `with` with capital W while in line 1 it is lower in the Help with Fees text